src/main/java/org/alfresco/rest/api/search/impl/ResultMapper.java

```diff
@@ -635,4 +642,31 @@ protected SolrJSONResultSet findSolrResultSet(ResultSet results)
 
         return null;
     }
+    public CollectionWithPagingInfo<TupleList> toCollectionWithPagingInfo(JSONArray docs, SearchSQLQuery searchQuery) throws JSONException
```

Comment:
searchQuery is not used in this method. No need to pass it as an argument.

Reply:
true, now removed.